howsthewaterfront/src/components/Footer.js

```diff
@@ -1,7 +1,74 @@
 import React from "react";
+import styled from "styled-components";
 
-const Footer = () => {
-  return <div>Footer</div>;
+const FooterDiv = styled.div`
+  display: flex;
+  width: 100%;
+  margin: 5% 0 2% 0;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  font-weight: 500;
+  font-family: Ubuntu;
+  font-size: 1.6rem;
+  color: #2d728f;
+
+  div {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    font-size: 1.6rem;
+
+    a {
+      text-decoration: none;
+      color: #2d728f;
+
+      :hover {
+        color: #26bcd7;
+      }
+    }
+  }
+
+  .line {
+    margin: 0 5px 0 5px;
+  }
+
+  @media (max-width: 894px) {
+    width: 70%;
+    flex-direction: column;
+
+    .copyright {
+      margin-bottom: 1%;
+    }
+
+    .first {
+      display: none;
+    }
+  }
+
+  @media (max-width: 576px) {
+    .copyright {
+      width: 103%;
+    }
+  }
+`;
+
+const CopyrightFooter = () => {
+  return (
+    <FooterDiv>
+      <p className="copyright">
+        Copyright 2019 How's The Water Inc. All Rights Reserved
```

Comment:
This looks good @jamesbieber but maybe we should use the copyright symbol instead of the word "copyright"? It looks a bit cleaner. That is only if everybody agrees, otherwise, it looks good to go!

Reply:
I'm fine with switching it to the '©' if everyone else is for it.